packages/data-context/package.json

```diff
@@ -27,7 +27,6 @@
     "dayjs": "^1.9.3",
     "dedent": "^0.7.0",
     "ejs": "^3.1.6",
-    "electron": "14.1.0",
```

Comment:
Should we not leave it as a `devDependency` here, since it's imported in `data-context`? (if only for types with `import type`)?

Reply:
I think this might have broke the `ELECTRON_RUN_AS_NODE` feature. docs: https://docs.cypress.io/guides/continuous-integration/introduction#Running-headless-tests-without-Xvfb

https://github.com/cypress-io/cypress/issues/23636